Sources/KlaviyoSwift/StateManagement.swift

```diff
@@ -16,103 +16,114 @@ import Foundation
 // Request flush interval in seconds.
 let CELLULAR_FLUSH_INTERVAL = 30.0
 let WIFI_FLUSH_INTERVAL = 10.0
+let MAX_QUEUE_SIZE = 200
+
+enum RetryInfo: Equatable {
+    case retry(Int) // Int is current count for first request
+    case retryWithBackoff(Int, Int, Int) // requestCount, totalRetryCount, currentBackoff
```

Comment:
would it make sense to label the enums associated values? 
Ex: `case retryWithBackoff(requestCount : Int, totalRetryCount: Int, currentBackoff: Int)`